src/ORM/Marshaller.php

```diff
@@ -167,6 +167,7 @@ public function one(array $data, array $options = [])
 
         $primaryKey = (array)$this->_table->primaryKey();
         $entityClass = $this->_table->entityClass();
+        /* @var Entity $entity */
```

Comment:
@markstory maybe, I think the function is using methods from Entity and not the interface, but I'll double check.